packages/presentation/lib/style/text_styles/styles.dart

```diff
@@ -24,12 +24,36 @@ TextStyle sfProSemiMedium12px50opacity = TextStyle(
   fontSize: 12,
 );
 
+TextStyle sfProSemiMedium18px = TextStyle(
+  color: Color(0xffFFFFFF),
+  fontFamily: 'SF Pro Text Medium',
+  fontSize: 18,
+);
+
+TextStyle sfProSemiMedium14pxBlue = TextStyle(
+  color: Color(0xff47CFFF),
+  fontFamily: 'SF Pro Text Medium',
```

Comment:
magic string
you should create private variable fe const String _fontFamily = 'SF Pro Text Semibold'; and use it